install_config/router/f5_router.adoc

```diff
@@ -143,8 +143,94 @@ link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos_ma
 Use the `--external-host-partition-path` flag when
 xref:deploying-the-f5-router[deploying the F5 router] to specify a partition
 path:
-====
+
 ----
 $ oadm router --external-host-partition-path=/OpenShift/zone1 ...
 ----
+
+[[setting-up-f5-native-integration-with-openshift]]
+== Setting Up F5 Native Integration
+
+[NOTE]
 ====
+This section reviews how to set up F5 native integration with {product-title}.
+The concepts of F5 appliance and {product-title} connection and data flow of F5
+native integration are discussed in the
+xref:../../architecture/core_concepts/routes.adoc#architecture-f5-native-integration[F5
+Native Integration] section of the Routes topic.
+====
+
+With new native integration of F5 with {product-title}, you do not need to
+configure a ramp node for F5 to be able to reach the pods on the overlay network
+as created by OpenShift SDN.
+
+The F5 controller pod needs to be launched with enough information that it can
+successfully connect to pods directly.
```

Comment:
"...with enough information _so_ that it can..."

Also, "...that it can successfully _directly_ connect to pods." There's a rule somewhere about keeping adverbs close to the words they're modifying, but I don't think it's anywhere in IBM...